gradle/publishing.gradle

```diff
@@ -1,7 +1,10 @@
 apply plugin: 'maven-publish'
-apply plugin: 'nebula.nebula-bintray'
+apply plugin: 'nebula.nebula-bintray-publishing'
+apply plugin: 'com.jfrog.artifactory'
```

Comment:
This is needed since nebula bintray is apply the artifactory plugin to sub-projects